src/components/ReleaseService/ReleasePlan/ReleasePlanForm/ReleasePlanFormPage.tsx

```diff
@@ -0,0 +1,133 @@
+        // eslint-disable-next-line camelcase
+        release_plan_name: releasePlan.metadata.name,
+        // eslint-disable-next-line camelcase
+        target_workspace: releasePlan.spec.target,
+        app_name: releasePlan.spec.application,
+        workspace,
+      });
+    } else {
+      track(TrackEvents.ButtonClicked, {
+        link_name: 'add-release-plan-submit',
+        workspace,
+      });
+    }
+    try {
+      edit
+        ? await editReleasePlan(releasePlan, values, workspace, true)
+        : await createReleasePlan(values, namespace, workspace, true);
+      const newReleasePlan = edit
+        ? await editReleasePlan(releasePlan, values, workspace)
+        : await createReleasePlan(values, namespace, workspace);
+      track(edit ? 'Release plan edited' : 'Release plan created', {
+        // eslint-disable-next-line camelcase
+        release_plan_name: newReleasePlan.metadata.name,
+        // eslint-disable-next-line camelcase
+        target_workspace: newReleasePlan.spec.target,
+        app_name: newReleasePlan.spec.application,
+        workspace,
+      });
+      navigate('/application-pipeline/release');
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.warn('Error while submitting integration test:', e);
+      setSubmitting(false);
+      setStatus({ submitError: e.message });
+    }
+  };
+
+  const handleReset = () => {
+    if (edit) {
+      track(TrackEvents.ButtonClicked, {
+        link_name: 'edit-release-plan-leave',
+        // eslint-disable-next-line camelcase
+        release_plan_name: releasePlan.metadata.name,
+        // eslint-disable-next-line camelcase
+        target_workspace: releasePlan.spec.target,
+        app_name: releasePlan.spec.application,
+        workspace,
+      });
+    } else {
+      track(TrackEvents.ButtonClicked, {
+        link_name: 'add-release-plan-leave',
+        workspace,
+      });
+    }
+    navigate('/application-pipeline/release');
+  };
+
+  const initialValues: ReleasePlanFormValues = {
+    name: releasePlan?.metadata?.name ?? '',
+    application: releasePlan?.spec?.application ?? '',
+    autoRelease: releasePlan?.metadata?.labels?.[ReleasePlanLabel.AUTO_RELEASE] === 'true' ?? false,
+    standingAttribution:
+      releasePlan?.metadata?.labels?.[ReleasePlanLabel.STANDING_ATTRIBUTION] === 'true' ?? false,
+    releasePipelineLocation: releasePlan?.spec?.target
+      ? releasePlan.spec.target === workspace
+        ? ReleasePipelineLocation.current
+        : ReleasePipelineLocation.target
+      : undefined,
+    serviceAccount: releasePlan?.spec?.serviceAccount ?? '',
+    target: releasePlan?.spec?.target ?? '',
+    data: releasePlan?.spec?.data ?? '',
+    params: releasePlanFormParams(releasePlan),
+    labels: releasePlan?.metadata?.labels
+      ? Object.entries(releasePlan?.metadata?.labels).map(([key, value]) => ({ key, value }))
+      : [{ key: '', value: '' }],
```

Comment:
Please remove this empty key, value label. This is causing dry run error
<img width="915" alt="image" src="https://github.com/openshift/hac-dev/assets/9278015/c1282dc5-edb6-4a92-b110-7f4d933d91b7">

Reply:
Just pulled latest again it is fixed